algorithms/dqn/agent.py

```diff
@@ -235,6 +260,7 @@ def train(self):
             score += reward[0]
             i_episode_prev = i_episode
             i_episode += done.sum()
+            self.i_episode = i_episode
 
             if (i_episode // self.args.save_period) != (
```

Comment:
should change "!=" to ">" ?

Reply:
doesn't really matter unless i_episode decreases